### DISCUSSION TITLE
Line break for error messages

### DISCUSSION THREAD

**Discussion by jschoone** [hrsh7th/nvim-cmp#1061] [Q&A] [answered]
Hi,
I installed nvim-cmp yesterday the first time and it looks good so far.
For Go I used the [gopls example](https://github.com/hrsh7th/nvim-cmp/wiki/Language-Server-Specific-Samples#golang-gopls) which also works good, but I have the probleme the error messages don't fit the screen. See attached screenshot.

I tried to search for a solution on my own, but I have no idea what I need to search and if this is nvim or nvim-cmp or gopls. Can someone shed some light on this? A keyword for what I should look for would be enough.

Thank you

![image](https://user-images.githubusercontent.com/6106846/177027827-53df8f4b-538c-416a-b1f9-eeba6897d733.png)

**Comment by fitrh** [comment, marked as answer]
It's not `cmp`, it's `vim.diagnostic`, see `:h vim.diagnostic`

**Reply by jschoone** [reply]
Thank you, that's what I looked for. Now I can get further. :)
